packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -184,6 +219,21 @@ export const CreateAlertDefinition = () => {
             }}
             sx={{ display: 'flex', justifyContent: 'flex-end' }}
           />
+          {openAddNotification && (
+            <Drawer
+              onClose={onExitNotifications}
+              open={openAddNotification}
+              title="Add Notification Channel"
+            >
+              <AddNotificationChannel
+                isNotificationChannelsError={false}
+                isNotificationChannelsLoading={false}
+                onCancel={onExitNotifications}
+                onSubmitAddNotification={onSubmitAddNotification}
+                templateData={notificationChannelFactory.buildList(2)}
+              />
+            </Drawer>
+          )}
```

Comment:
Let's remove this conditional. It breaks the drawer open/close animation

```suggestion
          <Drawer
            onClose={onExitNotifications}
            open={openAddNotification}
            title="Add Notification Channel"
          >
            <AddNotificationChannel
              isNotificationChannelsError={false}
              isNotificationChannelsLoading={false}
              onCancel={onExitNotifications}
              onSubmitAddNotification={onSubmitAddNotification}
              templateData={notificationChannelFactory.buildList(2)}
            />
          </Drawer>
```

The Drawer component should be smart enough to not render its children when open is `false` so performance should still be okay

Reply:
Understood, made the suggested chage